src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Shape {
+
+    private Random rand;
+    private StringBuilder sb = new StringBuilder();
+    private double roundness; // Окружность
+
+    public Circle(String name) {
+        super(name);
+        rand = new Random();
+        this.roundness = rand.nextInt(50);
+    }
+
+    @Override
+    public double calculateArea() {
+        return (int) (radius() * Math.PI);
```

Comment:
why cast to int?

Reply:
I wanted to round a digit. Fixed it. Now I use a method Math.round();